spec/classes/datadog_agent_integrations_nginx_spec.rb

```diff
@@ -43,11 +43,25 @@
               'nginx_status_url' => 'http://foo.bar:1941/check',
               'tags' => %w{foo bar baz}
             }
+          ],
+          logs: [
+            {
+              'type' => 'file',
+              'path' => '/var/log/nginx/access.log',
+              'service' => 'my.nginx.service',
```

Comment:
This should be `nginx` based on: https://github.com/DataDog/integrations-core/blob/7e794127f1312d6830880f35bee54ae2668f25f6/nginx/datadog_checks/nginx/data/conf.yaml.example#L80.